docs/guides/on-demand-guides/index.mdx

```diff
@@ -3,4 +3,740 @@ title: On Demand
+    "status": "waiting",
+    "name": "Waterfall"
+  },
+  "task": {
+    "id": "58a9b4dc-89d8-40f5-8372-cacaec49024b",
+    "createdAt": 1652901725741,
+    "type": "import",
+    "outputAssetId": "4c21895e-2488-4c0a-9377-3369ff586ddd",
+    "userId": "eada599f-3d58-499b-ba24-c7f3faf988de",
+    "params": {
+      "import": {
+        "uploadedObjectKey": "directUpload/4c21k7z2tjcdg8wt/source"
+      }
+    },
+    "status": {
+      "phase": "pending",
+      "updatedAt": 1652901725741
+    }
+  }
+}
+```
+
+2. Upload the asset to the generated URL
+
+Get the “url”: “https://origin.livepeer.studio/api/assets/…“ from the response.
+
+Check the status of the generated url.
+
+https://livepeer.studio/api/asset/$ASSET_ID/ with provided asset.id
+
+### Request
+
+
+```JavaScript
+{
+  "id": "4c21895e-2488-4c0a-9377-3369ff586ddd",
+  "name": "Waterfall",
+  "status": "waiting",
+  "userId": "eada599f-3d58-499b-ba24-c7f3faf988de",
+  "createdAt": 1652895834393,
+  "playbackId": "4c21k7z2tjcdg8wt"
+}
+```
+
+### Upload the video
+
+### Request
+
+```JavaScript
+
+const uploadAsset = async () => {
+
+const response = await axios
+  .put("${url}", {
+    headers: {
+      Authorization: `Bearer ${process.env.LP_API_KEY}`
+      "Content-Type": "video/mp4"
+    },
+    data: fs.createReadStream(path)
+  });
+    console.log(JSON.stringify(response.data));
+  };
+```
+
+### Response
+
+When the video is uploaded, the result should return a 200 response
+
+```JSON
+{
+  "id": "4c21895e-2488-4c0a-9377-3369ff586ddd",
+  "hash": [
+    {
+      "hash": "6dad0a8e7a92d537c6305fe6ae1e1881",
+      "algorithm": "md5"
+    },
+    {
+      "hash": "6132f4e43472b56b4047b7ad018147056c0c2dba9306cd04504e9c8208e72257",
+      "algorithm": "sha256"
+    }
+  ],
+  "name": "Waterfall",
+  "size": 32407849,
+  "status": "ready",
+  "userId": "eada599f-3d58-499b-ba24-c7f3faf988de",
+  "createdAt": 1652901725324,
+  "updatedAt": 1652901778931,
+  "videoSpec": {
+    "format": "mp4",
+    "tracks": [
+      {
+        "fps": 23.976023976023978,
+        "type": "video",
+        "codec": "h264",
+        "width": 4096,
+        "height": 2048,
+        "bitrate": 21324821,
+        "duration": 12.011667,
+        "pixelFormat": "yuv420p"
+      },
+      {
+        "type": "audio",
+        "codec": "aac",
+        "bitrate": 253375,
+        "channels": 2,
+        "duration": 12.031667,
+        "sampleRate": 48000
+      }
+    ],
+    "duration": 12.031667
+  },
+  "playbackId": "4c21k7z2tjcdg8wt",
+  "playbackUrl": "https://livepeercdn.com/recordings/54aeff83-404c-490a-9afa-99296a932401/index.m3u8",
+  "downloadUrl": "https://livepeercdn.com/asset/4c21k7z2tjcdg8wt/video"
+}
+```
+
+
+## List Assets
+
+When listing assets, look for the status in the response body, it should return status:“ready”. If the response returns status:“waiting”, then wait a little longer and repeat this step until the status is ready before moving to the exporting step.
```

Comment:
When listing assets, look for the status in the response body, it should return `status:“ready”`. If the response returns `status:“waiting”`, then wait a little longer and repeat this step until the status is ready before moving to the exporting step.